.github/workflows/website.yml

```diff
@@ -0,0 +1,55 @@
+name: Website
+on:
+  push:
+    branches: gh-pages
+  pull_request: []
+jobs:
+  build-website:
+    name: Build lesson website
+    if: github.repository != 'carpentries/styles'
```

Comment:
I'm wondering if it'd be worth to change the logic here, so that on `carpentries/styles` we create an extra step to run `python ./bin/lesson_initialize.py` before running the `make lesson-check`. There might be a few other things to do manually on a newly initialized lesson before it passes the checks.

Reply:
that would give us an additional check in this repo to make sure that lessons can be initialized without any issues (and ideally that the newly initialized lesson passes the checks).

Reply:
Wouldn't it be better to do this as a separate GH Action "New Lesson Setup" or as a second job in "template.yml"?